test/runtests.jl

```diff
@@ -77,3 +86,55 @@ end
         @test separation(c3, c6) ≈ separation(c6, c3) ≈ 1
     end
 end
+
+@testset "hr2rad" begin
+    @test hr2rad(0) == 0
+    @test hr2rad(24) ≈ 2π
+    @test hr2rad(12.0) ≈ π
+    # https://github.com/JuliaAstro/SkyCoords.jl/pull/29#discussion_r354602964
+    @test 1 -
+          hr2rad(big(12)) /
+          π == -1.727233711018888925077270372560079914223200072887256277004740694033718360632485e-77
```

Comment:
Here I'd simply test
```julia
hr2rad(big(12)) ≈ π
```
I don't think it's a good idea to check exact equality when floating point numbers are involved.  With the original definition this test would fail, which is good:
```julia
julia> hr2rad(h::Number) = 2π/24 * h
hr2rad (generic function with 1 method)

julia> hr2rad(big(12)) ≈ π
false
```